### ISSUE TITLE
Dead keys do not work

### ISSUE THREAD

**Issue by nenad-novak** [gwsw/less#352]
Croation keyboard requires AltGr+'3' followed by 'space' key combination to get '^' char.
To get 'ü' charchter, AltGr+''' followed by 'u' is required.

Here is a fix win32_kbhit function to allow dead key combinations.

	} while (ip.EventType != KEY_EVENT ||
		ip.Event.KeyEvent.bKeyDown != TRUE ||
		(ip.Event.KeyEvent.wVirtualScanCode == 0 && ip.Event.KeyEvent.uChar.UnicodeChar == 0) ||
		(
		   ((ip.Event.KeyEvent.dwControlKeyState & (RIGHT_ALT_PRESSED | LEFT_CTRL_PRESSED)) == (RIGHT_ALT_PRESSED | LEFT_CTRL_PRESSED))
			 &&
		   (ip.Event.KeyEvent.uChar.UnicodeChar == 0)
		) ||
		ip.Event.KeyEvent.wVirtualKeyCode == VK_KANJI ||
		ip.Event.KeyEvent.wVirtualKeyCode == VK_SHIFT ||
		ip.Event.KeyEvent.wVirtualKeyCode == VK_CONTROL ||
		ip.Event.KeyEvent.wVirtualKeyCode == VK_MENU);

**Comment by gwsw** [comment]
@adoxa Hi Jason, do you see any problem with this?
```
diff --git a/screen.c b/screen.c
index 3aa5da2..fb43791 100644
--- a/screen.c
+++ b/screen.c
@@ -2849,6 +2849,7 @@ public int win32_kbhit(void)
    } while (ip.EventType != KEY_EVENT ||
        ip.Event.KeyEvent.bKeyDown != TRUE ||
        (ip.Event.KeyEvent.wVirtualScanCode == 0 && ip.Event.KeyEvent.uChar.UnicodeChar == 0) ||
+       ((ip.Event.KeyEvent.dwControlKeyState & (RIGHT_ALT_PRESSED|LEFT_CTRL_PRESSED)) == (RIGHT_ALT_PRESSED|LEFT_CTRL_PRESSED) && ip.Event.KeyEvent.uChar.UnicodeChar == 0) ||
        ip.Event.KeyEvent.wVirtualKeyCode == VK_KANJI ||
        ip.Event.KeyEvent.wVirtualKeyCode == VK_SHIFT ||
        ip.Event.KeyEvent.wVirtualKeyCode == VK_CONTROL ||
```

**Comment by adoxa** [comment]
Nope.

**Comment by gwsw** [comment]
Fixed in aa68f38b536f8c3ef75dc21be37e25ffbb6e6eaf.
